src/js/components/Grid/doc.js

```diff
@@ -1,6 +1,6 @@
-import { describe, PropTypes } from 'react-desc';
+import {describe, PropTypes} from 'react-desc';
```

Comment:
I think we will like to keep the spaces and be consistent across the files.

Reply:
I can fix this. What linter do you use? somehow my WebStorm linter does this automatically.

Reply:
prettier. and it actually runs as a pre-commit hook.